.rubocop.yml

```diff
@@ -0,0 +1,9 @@
+AllCops:
+  Include:
+    - '**/Berksfile'
+    - '**/Cheffile'
+  Exclude:
+    - 'metadata.rb'
+
+Metrics/AbcSize:
+  Max: 9001
```

Comment:
I will rewrite the function in a future PR and reduce this back to default. But @armiller and I agreed to punt for now.